openstack/blockstorage/extensions/backups/doc.go

```diff
@@ -71,5 +71,47 @@ Example to Delete a Backup
 	if err != nil {
 		panic(err)
 	}
+
+Example to Export a Backup
+
+	export, err := backups.Export(client, "uuid").Extract()
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println(export)
+
+Example to Import a Backup
+
+	availabilityZone := "region1b"
+	backupURL, _ := json.Marshal(backups.ImportBackup{
+		ID:               "d32019d3-bc6e-4319-9c1d-6722fc136a22",
+		Status:           "available",
+		AvailabilityZone: &availabilityZone,
+		VolumeID:         "cf9bc6fa-c5bc-41f6-bc4e-6e76c0bea959",
+		UpdatedAt:        time.Date(2020, 3, 11, 19, 29, 8, 0, time.UTC),
+		Host:             "cinder-backup-host1",
+		UserID:           "93514e04-a026-4f60-8176-395c859501dd",
+		ServiceMetadata:  "volume_cf9bc6fa-c5bc-41f6-bc4e-6e76c0bea959/20200311192855/az_regionb_backup_b87bb1e5-0d4e-445e-a548-5ae742562bac",
+		Size:             1,
+		ObjectCount:      2,
+		Container:        "my-test-backup",
+		Service:          "cinder.backup.drivers.swift.SwiftBackupDriver",
+		CreatedAt:        time.Date(2020, 3, 11, 19, 25, 24, 0, time.UTC),
+		DataTimestamp:    time.Date(2020, 3, 11, 19, 25, 24, 0, time.UTC),
+		ProjectID:        "14f1c1f5d12b4755b94edef78ff8b325",
+	})
+
+	options := backups.ImportOpts{
+		BackupService: "cinder.backup.drivers.swift.SwiftBackupDriver",
+		BackupURL:     backupURL,
```

Comment:
This needs to be base64 encoded, right?

Reply:
If you provide `[]byte` to a json encoder, it automatically encodes/decodes it to base64:

> Array and slice values encode as JSON arrays, except that []byte encodes as a base64-encoded string, and a nil slice encodes as the null JSON value.

https://golang.org/pkg/encoding/json/

Reply:
Well I learned something today 🙂 Thanks!

Reply:
I learned this, when I could not understand why I get base64 string in acceptance tests. It appeared that I forgot to place `w.Header().Add("Content-Type", "application/json")` into a mock list func: https://github.com/gophercloud/gophercloud/blob/965816e95e604e19c77b1a086c9737164dc3a4c5/openstack/blockstorage/extensions/volumetransfers/testing/fixtures.go#L199
Just try to comment this string out and run tests ;)